core23.md

```diff
@@ -17,22 +17,22 @@ Implementation of the JavaServer™ Faces (JSF) 2.3 specification.
 |-----------------|------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------|
-| binary (tar.gz) | [myfaces-core-assembly-2.3.10-bin.tar.gz](https://www.apache.org/dyn/closer.lua/myfaces/binaries/myfaces-core-assembly-2.3.10-bin.tar.gz) | [myfaces-core-assembly-2.3.10-bin.tar.gz.sha512](https://downloads.apache.org/myfaces/binaries/myfaces-core-assembly-2.3.10-bin.tar.gz.sha512) | [myfaces-core-assembly-2.3.10-bin.tar.gz.asc](https://downloads.apache.org/myfaces/binaries/myfaces-core-assembly-2.3.10-bin.tar.gz.asc) |
-| binary (zip)    | [myfaces-core-assembly-2.3.10-bin.zip](https://www.apache.org/dyn/closer.lua/myfaces/binaries/myfaces-core-assembly-2.3.10-bin.zip)       | [myfaces-core-assembly-2.3.10-bin.zip.sha512](https://downloads.apache.org/myfaces/binaries/myfaces-core-assembly-2.3.10-bin.zip.sha512)       | [myfaces-core-assembly-2.3.10-bin.zip.asc](https://downloads.apache.org/myfaces/binaries/myfaces-core-assembly-2.3.10-bin.zip.asc)       |
-| source (tar.gz) | [myfaces-core-assembly-2.3.10-src.tar.gz](https://www.apache.org/dyn/closer.lua/myfaces/source/myfaces-core-assembly-2.3.10-src.tar.gz)   | [myfaces-core-assembly-2.3.10-src.tar.gz.sha512](https://downloads.apache.org/myfaces/source/myfaces-core-assembly-2.3.10-src.tar.gz.sha512)   | [myfaces-core-assembly-2.3.10-src.tar.gz.asc](https://downloads.apache.org/myfaces/source/myfaces-core-assembly-2.3.10-src.tar.gz.asc)   |
-| source (zip)    | [myfaces-core-assembly-2.3.10-src.zip](https://www.apache.org/dyn/closer.lua/myfaces/source/myfaces-core-assembly-2.3.10-src.zip)         | [myfaces-core-assembly-2.3.10-src.zip.sha512](https://downloads.apache.org/myfaces/source/myfaces-core-assembly-2.3.10-src.zip.sha512)         | [myfaces-core-assembly-2.3.10-src.zip.asc](https://downloads.apache.org/myfaces/source/myfaces-core-assembly-2.3.10-src.zip.asc)         |
+| binary (tar.gz) | [myfaces-core-assembly-2.3.11-bin.tar.gz](https://www.apache.org/dyn/closer.lua/myfaces/binaries/myfaces-core-assembly-2.3.11-bin.tar.gz) | [myfaces-core-assembly-2.3.11-bin.tar.gz.sha512](https://downloads.apache.org/myfaces/binaries/myfaces-core-assembly-2.3.11-bin.tar.gz.sha512) | [myfaces-core-assembly-2.3.11-bin.tar.gz.asc](https://downloads.apache.org/myfaces/binaries/myfaces-core-assembly-2.3.11-bin.tar.gz.asc) |
```

Comment:
None of these links are currently working for me.

Reply:
I don't see 2.3.11 here for example: https://archive.apache.org/dist/myfaces/source/

Reply:
Files are uploaded -- the links should work soon.

Reply:
Though, I'd rather not link directly since the currently linked page contains verification instructions.